zerver/lib/notifications.py

```diff
@@ -492,14 +494,14 @@ def enqueue_welcome_emails(email, name):
                                          tags=["followup-emails"],
                                          sender=sender)
     # Send day 2 email
-    tomorrow = datetime.datetime.utcnow() + datetime.timedelta(hours=24)
+    tomorrow = timezone.now() + datetime.timedelta(hours=24)
     # 11 AM EDT
-    tomorrow_morning = datetime.datetime(tomorrow.year, tomorrow.month, tomorrow.day, 15, 0)
-    assert(datetime.datetime.utcnow() < tomorrow_morning)
+    tomorrow_morning = datetime.datetime(tomorrow.year, tomorrow.month, tomorrow.day, 15, 0, tzinfo=pytz.utc)
+    assert(timezone.now() < tomorrow_morning)
     send_local_email_template_with_delay([{'email': email, 'name': name}],
                                          "zerver/emails/followup/day2",
                                          template_payload,
-                                         tomorrow_morning - datetime.datetime.utcnow(),
+                                         tomorrow_morning - timezone.now(),
                                          tags=["followup-emails"],
```

Comment:
We want to change the behavior here anyway (time at which we send day 2 emails), so you can just drop your changes and I'll push something in a separate PR.